src/ceph.in

```diff
@@ -849,7 +849,11 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, ('mon', ''), ' '.join(childargs))
+
+    # implement "tell module help"
+    if len(childargs) >= 3 and childargs[0] == 'tell' and childargs[2] == 'help':
+        return do_extended_help(parser, childargs, childargs[1].split('.'), None)
```

Comment:
could you report an error if childarg[1] is neither `osd.[\d+]` nor `mon[a-z]+` or `mgr[a-z]+`, before passing it to `json_command()`? and if the component before `.` is not `osd` and the length of `childargs[1].split('.')` is 1, you might need to append an empty string to the end of it.

 i have
```
$ ./bin/ceph tell osd help
Traceback (most recent call last):
  File "./bin/ceph", line 1066, in <module>
    retval = main()
  File "./bin/ceph", line 860, in main
    return do_extended_help(parser, childargs, childargs[1].split('.'), None)
  File "./bin/ceph", line 307, in do_extended_help
    return help_for_target(target, partial)
  File "./bin/ceph", line 298, in help_for_target
    timeout=10)
  File "/var/ceph/ceph/src/pybind/ceph_argparse.py", line 1342, in json_command
    raise RuntimeError('"{0}": exception {1}'.format(argdict, e))
RuntimeError: "None": exception tuple index out of range
```
in my testbed.


partial fix looks like:
```py
        if len(target) == 1:
            if target == 'osd':
                print('Please pass osd.<n> instead as '
                      'the target of tell command', file=sys.stderr)
                return 1
            target.append('')
        return do_extended_help(parser, childargs, target, None)
```